modules/3d/test/test_fisheye.cpp

```diff
@@ -385,4 +385,32 @@ TEST_F(fisheyeTest, jacobians)
     CV_Assert (cv::norm(x2 - xpred) < 1e-10);
 }
 
+TEST_F(fisheyeTest, solvePnP)
```

Comment:
Don't mess tests order between branches to avoid further merge conflicts.

This test comes right before `TEST_F(fisheyeTest, undistortImage)` on 4.x branch:

https://github.com/opencv/opencv/blob/12aa0fe89819c02db53677c5749771a1007f623e/modules/calib3d/test/test_fisheye.cpp#L153-L175

This corresponds to the line 132 on 5.x branch.

---

If you want to fix order, then do that on 4.x branch first (or better perform reviews of origin PRs)